packages/vitest/src/node/plugins/globalSetup.ts

```diff
@@ -7,13 +9,25 @@ interface GlobalSetupFile {
   teardown?: Function
 }
 
-async function loadGlobalSetupFiles(server: ViteDevServer): Promise<GlobalSetupFile[]> {
+async function loadGlobalSetupFiles(ctx: Vitest): Promise<GlobalSetupFile[]> {
+  const node = ctx.vitenode
+  const server = ctx.server
+  const runner = new ViteNodeRunner({
+    root: server.config.root,
+    base: server.config.base,
+    fetchModule(id) {
+      return node.fetchModule(id)
+    },
+    resolveId(id, importer) {
+      return node.resolveId(id, importer)
+    },
+  })
```

Comment:
Could we also reuse the vite-node server?

Reply:
you mean the runner? The only 2 instances i found of that are in the [cli](https://github.com/vitest-dev/vitest/blob/568668ae747e3cbe162b7b13dc6e259655c6c313/packages/vite-node/src/cli.ts#L79) not exposed/sharable and in (execute)[https://github.com/vitest-dev/vitest/blob/4dbefb59cd673f5620266c882e50a1744d4fcc99/packages/vitest/src/node/execute.ts#L12] but thats wrapped in VitestRunner and we don't want/need mocking for globalSetup files :thinking:

Reply:
Ah, never mind, I misread :P